src/components/movielist/movielist.jsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react"
+import style from "./movielist.module.css"
+import { NavLink } from "react-router-dom"
+
+// API KEY passed into the API URL
+const api_key = "624685d1a2da244d659c85520d6ee611";
+const url = `https://api.themoviedb.org/3/movie/popular?api_key=${api_key}&language=en-US&page=1`;
+
+// Function to export the component
+export const MovieList = () => {
+    const [allmovies, setAllmovies] = useState([]);
+    // error 2 useState []
+
+    // Function to fetch the API - MovieList
+    useEffect(() => {
+
+        const fetchMoviesList = async () => {
+
+            const movieItems = document.querySelectorAll('.movieItem'); // hover effect mobile devices
+
+            movieItems.forEach(item => {
```

Comment:
I think it is great that you added the effect for mobiles!! I never heard of "touchstart"/"touch", and I will look it up!!